js/libs/keycloak-js/dist/keycloak.d.ts

```diff
@@ -332,6 +332,7 @@ export interface KeycloakProfile {
 	emailVerified?: boolean;
 	totp?: boolean;
 	createdTimestamp?: number;
+	attributes?: Record<string, any>;
```

Comment:
Perhaps `unknown` would be a better type for the value here?
```suggestion
	attributes?: Record<string, unknown>;
```